utilities/getProcessedImage.ts

```diff
@@ -25,6 +25,9 @@ export const getProcessedImage = (
 ): string => {
   if (!imageSrc) {
     return '';
+  // Leave the SVGs alone since they don't need to be processed
+  } else if (imageSrc.endsWith('.svg')) {
+    return getMaskedAsset(imageSrc);
```

Comment:
This was added on Tour because their logo is svg. It would be useful on Momentum too.

Reply:
@ericbakenhus Updating `getProcessedImage` utility here 😄